node/tests/TestUtilities.ts

```diff
@@ -1758,7 +1767,8 @@ export async function transactionTest(
     }
 
     baseTransaction.wait(1, 200);
-    responseData.push(["wait(1, 200)", 1]);
+    if (gte(version, "7.0.0")) responseData.push(["wait(1, 200)", 1]);
+    else responseData.push(["wait(1, 200)", 0]);
```

Comment:
lolwut, if `wait` returns 0, replica isn't in sync with master node

Reply:
If the command is sent as part of a [MULTI](https://redis.io/docs/latest/commands/multi/) transaction (since Redis 7.0, any context that does not allow blocking, such as inside scripts), the command does not block but instead just return ASAP the number of replicas that acknowledged the previous write commands.